lib/driver.js

```diff
@@ -104,11 +104,14 @@ const APP_EXTENSION = '.apk';
 
 
 class EspressoDriver extends BaseDriver {
-  constructor (opts = {}, shouldValidateCaps = true) {
+  constructor (opts = {}, shouldValidateCaps = true, driverArgs = {}) {
     // `shell` overwrites adb.shell, so remove
     delete opts.shell;
 
-    super(opts, shouldValidateCaps);
+    opts.reboot = helpers.parseReboot(driverArgs);
```

Comment:
are these parsers really needed? I would simply pass args as is

Reply:
i agree that we don't need parsers since these values are coming in as JSON converted to JS objects. the only reason we'd ever need parsers is if the value coming in from the JSON is not the final value. for example, if it's a CSV (but in that case we could make the value an array instead!), or a filepath